go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/dgraph-io/badger/v2 v2.2007.4
 	github.com/ef-ds/deque v1.0.4
 	github.com/ethereum/go-ethereum v1.13.10
-	github.com/fxamacker/cbor/v2 v2.4.1-0.20230228173756-c0c9f774e40c
+	github.com/fxamacker/cbor/v2 v2.6.0
```

Comment:
This points to a branch, https://github.com/fxamacker/cbor/compare/feature/stream-mode, I don't think we can just update to the latest version right? (We should maybe also do this in atree and Cadence)

Could we maybe merge this branch, or at least tag it?

Reply:
@fxamacker what do you recommend we use?

Reply:
> @fxamacker what do you recommend we use?

@peterargue For now, we can stay with feature/stream-mode branch https://github.com/fxamacker/cbor/commit/c0c9f774e40c because that is based on fxamacker/cbor v2.4.0 + stream-mode features.

[v2.5.0](https://github.com/fxamacker/cbor/releases/tag/v2.5.0) has a few error handling bugfixes/improvements that might cause compatibility issues for some edge cases mentioned in release notes.  So before upgrading to newer fxamacker/cbor version, it would be good to review the code using it.